test/Dialect/SSP/roundtrip.mlir

```diff
@@ -142,64 +166,80 @@ ssp.instance @mco_outgoing_delays of "ChainingProblem" {
 
 // CHECK: ssp.instance @multiple_oprs of "SharedOperatorsProblem" {
 // CHECK:   library {
-// CHECK:     operator_type @slowAdd [latency<3>, limit<2>]
-// CHECK:     operator_type @fastAdd [latency<1>, limit<1>]
+// CHECK:     operator_type @slowAdd [latency<3>]
+// CHECK:     operator_type @fastAdd [latency<1>]
 // CHECK:     operator_type @_0 [latency<0>]
 // CHECK:     operator_type @_1 [latency<1>]
 // CHECK:   }
+// CHECK:  resource {
+// CHECK:    resource_type @slowAdder [limit<1>]
+// CHECK:    resource_type @fastAdder [limit<1>]
+// CHECK:  }
 // CHECK:   graph {
-// CHECK:     %[[op_0:.*]] = operation<@slowAdd>() [t<0>]
-// CHECK:     %[[op_1:.*]] = operation<@slowAdd>() [t<1>]
-// CHECK:     %[[op_2:.*]] = operation<@fastAdd>() [t<0>]
-// CHECK:     %[[op_3:.*]] = operation<@slowAdd>() [t<1>]
-// CHECK:     %[[op_4:.*]] = operation<@fastAdd>() [t<1>]
+// CHECK:     %[[op_0:.*]] = operation<@slowAdd>() [rsrc<@slowAdder>, t<0>]
```

Comment:
I think the used rsrc should be a first class property of the operation, similar to how the operator type is.

Reply:
makes sense!